apps/electron-app/src/main/core/environment-manager.ts

```
@@ -0,0 +1,209 @@
+ * - Sentry initialization
+ * - Log level configuration
+ * - Development vs production setup
+ *
+ * Replaces scattered environment setup from main process.
+ */
+export class EnvironmentManager {
+  private initialized = false;
+
+  constructor() {
+    // Constructor is intentionally simple
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)